### PR TITLE
Migration Nominator Guides

### LINKED ISSUE #4125: Migration Nominator Guides in Maintain Section
There are still some sections here that can be deleted and redirected to the relevant support articles.

See [this comment](https://github.com/w3f/polkadot-wiki/issues/3762#issuecomment-1239408276)

Regarding the migration of the nominator guides I propose to delete the "Unbonding and Rebonding" page, put everything in the "Nominate" page with links to the support articles, and redirect the deleted page there.

Motivation: the unbonding page has outdated screenshots, everything is in the support articles. Tbh I think we can just leave one standalone page for "Nominator Guides" as all the detailed info is in the learn section.

### PR DESCRIPTION
**Pull request by filippoweb3**
See related issue

### CODE REVIEW

File: docs/maintain/maintain-guides-how-to-nominate-polkadot.md [hunk 1]
```diff
@@ -40,143 +41,42 @@ performs due diligence on the validators that they elect. When looking for valid
 nominator should pay attention to their own reward percentage for nominating a specific validator -
 as well as the risk that they bear of being slashed if the validator gets slashed.
 
-If you are a beginner, please watch the video below for detailed instructions
+If you are a beginner, please watch the video below for detailed instructions.
 
-[![Stake on Polkadot/Kusama](https://img.youtube.com/vi/FCXC0CDhyS4/0.jpg)](https://youtu.be/FCXC0CDhyS4)
+[![Stake on Polkadot/Kusama](https://img.youtube.com/vi/KTBUOFq0KO4/0.jpg)](https://youtu.be/KTBUOFq0KO4)
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-nominate-polkadot.md]: I changed it to Ross's tutorial. Remember to change it to your tutorial after it is published @filippoweb3

File: docs/maintain/maintain-guides-how-to-nominate-polkadot.md [hunk 2]
```diff
@@ -40,143 +41,42 @@ performs due diligence on the validators that they elect. When looking for valid
...
-The "Waiting" subsection ([link](https://polkadot.js.org/apps/#/staking/waiting)) lists all pending
-validators that are awaiting more nominations to enter the active validator set. Validators will
-stay in the waiting queue until they have enough DOT backing them (as allocated through the
-[Phragmén election mechanism](../learn/learn-phragmen.md)). It is possible validator can remain in
-the queue for a very long time if they never get enough backing.
-
-The "Validator Stats" subsection ([link](https://polkadot.js.org/apps/#/staking/query)) allows you
-to query a validator's stash address and see historical charts on era points, elected stake,
-rewards, and slashes.
+:::info Support Articles
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-nominate-polkadot.md]:
```suggestion
:::info Using Polkadot-JS UI as a Nominator
```

File: docs/maintain/maintain-guides-how-to-nominate-polkadot.md [hunk 3]
```diff
@@ -40,143 +41,42 @@ performs due diligence on the validators that they elect. When looking for valid
...
 
-Pick "Account actions", then click the "+ Nominator" button.
+If you need to use the Polkadot-JS UI, clicking on the topics below will take you to the relevant support articles.
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-nominate-polkadot.md]:
```suggestion
Here is the list of basic nominator actions that can be performed using the Polkadot-JS UI.
```

File: docs/maintain/maintain-guides-how-to-nominate-polkadot.md [hunk 4]
```diff
@@ -40,143 +41,42 @@ performs due diligence on the validators that they elect. When looking for valid
...
-
-![Payout account selection dropdown with the custom account option highlighted](../assets/payout/01.png)
-
-:::note
-
-These concepts have been further explained in Polkadot's
-[UI Walkthrough Video](https://youtu.be/FCXC0CDhyS4?t=219)
+- [How to Bond Tokens and Nominate](https://support.polkadot.network/support/solutions/articles/65000168057-polkadot-js-ui-how-do-i-stake-nominate-on-polkadot-)
+- [How to Select Validators](https://support.polkadot.network/support/solutions/articles/65000150130-how-do-i-know-which-validators-to-choose-)
+- [How to Monitor the Bags List](https://support.polkadot.network/support/solutions/articles/65000181018-i-have-more-than-the-minimum-bonded-but-i-m-not-getting-rewards)
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-nominate-polkadot.md]: Probably need to move this article to a different location in the document. 
```suggestion
```

File: docs/maintain/maintain-guides-how-to-nominate-polkadot.md [hunk 5]
```diff
@@ -258,9 +158,10 @@ polkadot-js-api --seed "MNEMONIC_PHRASE" tx.staking.nominate '["VALIDATOR_ADDRES
 polkadot-js-api --seed "xxxx xxxxx xxxx xxxxx" tx.staking.nominate '["CmD9vaMYoiKe7HiFnfkftwvhKbxN9bhyjcDrfFRGbifJEG8","E457XaKbj2yTB2URy8N4UuzmyuFRkcdxYs67UvSgVr7HyFb"]' --ws wss://rpc.polkadot.io
 ```
 
-After a few seconds, you should see the hash of the transaction and if you would like to verify the
+After a few seconds, you should see the hash of the transaction, and if you would like to verify the
 nomination status, you can check that on the Polkadot-JS UI as well.
 
 [validators]: maintain-guides-how-to-validate-polkadot.md
-[keys]: ../learn/learn-keys.md###"controller"-and-"stash"-keys
+[keys]: ../learn/learn-keys.md#keys
```
Comment (profradha) [inline docs/maintain/maintain-guides-how-to-nominate-polkadot.md]: As the "keys" section is moved to cryptography page

```suggestion
[keys]: ../learn/learn-cryptography.md#keys
```

**Review by DrW3RK — COMMENTED** [review]
Added a few minor fixes. I think this doc would benefit from a section that discusses the existence of nomination pools and how the pools differ from the direct nomination. What do you think?

File: docs/maintain/maintain-guides-how-to-nominate-polkadot.md (hunk 1 above)
Comment (filippoweb3) [reply docs/maintain/maintain-guides-how-to-nominate-polkadot.md]: Done

File: docs/maintain/maintain-guides-how-to-nominate-polkadot.md (hunk 4 above)
Comment (filippoweb3) [reply docs/maintain/maintain-guides-how-to-nominate-polkadot.md]: Yep, created a section for this

**Review by DrW3RK — APPROVED** [review]
Looks good! Thanks @filippoweb3

**Comment by DrW3RK** [comment]
You can merge this and then work on the comparison on a separate PR for convenience

**Pull request merged**